src/rust-crypto/rust-crypto.ts

```diff
@@ -1466,6 +1466,7 @@ export class RustCrypto extends TypedEventEmitter<RustCryptoEvents, CryptoEventH
         await this.backupManager.deleteAllKeyBackupVersions();
 
         this.deleteSecretStorage();
+        await this.deleteSecretStorage();
```

Comment:
Can you update the test so this would be caught by it please?

Reply:
This is difficult because we are using a mocked SecretStorage which is way faster than the real execution. Meaning that even by adding `jest.fn().mockResolvedValue(undefined)` to `store` or `setDefaultKeyId`, the test passes.